packages/protobuf-test/typescript/tsconfig.4_7_4.json

```
@@ -12,6 +12,8 @@
     // compatible, set the skipLibCheck flag to false explicitly.
     "skipLibCheck": false,
     // To test forward-compatibility, set to NodeNext
-    "moduleResolution": "NodeNext"
+    "moduleResolution": "NodeNext",
+    "declaration": true,
+    "declarationMap": true
```

Can you add a comment here explaining why we set the option? 

```suggestion
    // Certain errors are only triggered by actually emitting declaration files, see https://github.com/bufbuild/protobuf-es/pull/398
    "declaration": true,
    "declarationMap": true
```

Please add the same comment to the other config files.